src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb

```diff
@@ -34,7 +34,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Optional path As String = "",
```

Comment:
Should we remove the optional parameters from this?

Reply:
Ah, yes. Good catch.